assignments/arrays.js

```diff
@@ -65,21 +65,45 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+console.log(inventory[32])
```

Comment:
Instead of logging the object, please log the car's year, make, and model as in the example above.